src/main/java/Duke.java

```diff
@@ -1,10 +1,230 @@
+            String current = sc.nextLine();
+            String[] taskInfo = current.split("/");
+            Task task = null;
+            switch (taskInfo[0]) {
+                case "T":
+                    task = new ToDo(taskInfo[2]);
+                    break;
+                case "D":
+                    task = new Deadline(taskInfo[2], taskInfo[3]);
+                    break;
+                case "E":
+                    task = new Event(taskInfo[2], taskInfo[3], taskInfo[4]);
+                    break;
+            }
+            if (task != null) {
+                if (taskInfo[1].equals("X")) {
+                    task.setDone();
+                }
+                tasks.add(task);
+            }
+        }
+    }
+
+    private static void resetTasksFile() {
+        try {
+            FileWriter fw = new FileWriter(storedTasksPath, false);
+            fw.write("");
+            fw.close();
+        } catch (IOException e) {
+            System.out.println("Something went wromg: " + e.getMessage());
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        File directory = new File("./data");
+        if (!directory.exists()) {
+            directory.mkdir();
+        }
+
+        File file = new File(storedTasksPath);
+        if (!file.exists()) {
+            try {
+                file.createNewFile();
+            } catch (IOException e) {
+                System.out.println("Error while creating file: " + e.getMessage());
+            }
+        }
+
+        Scanner fileScanner = null;
+        try {
+            fileScanner = new Scanner(file);
+        } catch (FileNotFoundException e) {
+            System.out.println("Stored tasks file does not exists: " + e.getMessage());
+        }
+
+        if (fileScanner != null) {
+            loadTasks(fileScanner, tasks);
+            resetTasksFile();
+        }
+
+        System.out.println("Hello! I'm Blob.\nWhat can I do for you?\n");
+        String message = sc.nextLine();
+
+        while (!message.equalsIgnoreCase("bye")) {
+            if (message.equalsIgnoreCase("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < tasks.size(); i++) {
+                    System.out.println((i + 1) + ". " + tasks.get(i));
+                }
+                System.out.println();
+            } else if (message.split(" ")[0].equalsIgnoreCase("mark")) {
+                try {
+                    String number = message.split(" ")[1];
+                    int n = Integer.parseInt(number);
```

Comment:
Perhaps more intuitive variable names here to differentiate between number and n?